packages/pwa-kit-dev/src/ssr/server/test_fixtures/node_modules/another-extension/setup-server.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* istanbul ignore next */
+const extension = ({app}) => {
+    app.get('/another-extension', (req, res) => {
+        res.send('test')
+    })
+}
+
+module.exports = {
+    default: extension
+}
```

Comment:
I noticed that all your test extensions are using common js export style. This isn't something that we normally do throughout our mono-repo, is it possible to have a text extension that user ex6 module exports?

Reply:
done!

we have 2 sets of tests, `pwa-kit-dev` and `pwa-kit-runtime`, i fixed the ones in `pwa-kit-dev` and the `pwa-kit-runtime` tests still have the commonjs exports since that's the built format on prod.